collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/CustomDatePickerDialog.java

```diff
@@ -210,6 +215,16 @@ protected void setUpYearPicker(int year, int minSupportedYear, int maxSupportedY
         yearPicker.setValue(year);
     }
 
+    protected void monthUpdated() {
```

Comment:
Both methods `methodUpdated` and `yearUpdated` have exactly the same bodies. One of the method can be dropped and called generically `dateUpdated`. What do you think?

Reply:
It's not possible because sometimes we would need to override for example just `monthUpdated()` or `yearUpdated()` method like it is in `MyanmarDatePickerDialog` class.